app/src/App.tsx

```diff
@@ -23,132 +22,144 @@ import { Guest } from './models'
-                      <Route exact path="/demo" component={Demo} />
-                      <Route path="/about" component={AboutPage} />
-                      <Route path="/profile" component={ProfileEditPage} />
-                      <Route path="/admin/guests" component={AdminView} />
-                      <Route
-                        path="/admin/guest/:id"
-                        component={AdminGuestView}
-                      />
-                      <Route
-                        path="/guests/:guestId/matches/:hostId"
-                        component={HostProfilePage}
-                      />
-                      <Route
-                        path="/guests/:id"
-                        component={GuestProfilePage}
-                      />
-                      <Route
-                        path="/admin/hosts"
-                        component={AllHosts}
-                      />
-                      <Route
-                        exact path="/profileselection/:id"
-                        component={ProfileSelection}
-                      />
-                      <Route
-                        exact path="/host/:id"
-                        component={HostQuestionsPage}
-                      />
-                      <Route
-                        exact path="/guest/:id"
-                        component={GuestQuestionsPage}
-                      />
-                      <FourOhFour />
-                    </Switch>
-                  </React.Fragment>
+                  </AppStyle.Holder>
+                  <AppStyle.Holder>
+                    <span>Hello, {(user && user.name) || 'User'}</span>
+                  </AppStyle.Holder>
+                  <AppStyle.Holder>
+                    <span onClick={logoutClick}>
+                      Logout
+                    </span>
+                  </AppStyle.Holder>
+                </AppStyle.FlexHolder>
+                <React.Fragment>
+                  <Switch>
+                    <Route exact path="/" component={AboutPage} />
+                    <Route exact path="/demo" component={Demo} />
+                    <Route path="/about" component={AboutPage} />
+                    <Route path="/profile" component={ProfileEditPage} />
+                    <Route path="/admin/guests" component={AdminView} />
+                    <Route
+                      path="/admin/guest/:id"
+                      component={AdminGuestView}
+                    />
+                    <Route
+                      path="/guests/:guestId/matches/:hostId"
+                      component={HostProfilePage}
+                    />
+                    <Route
+                      path="/guests/:id"
+                      component={GuestProfilePage}
+                    />
+                    <Route
+                      path="/admin/hosts"
+                      component={AllHosts}
+                    />
+                    <Route
```

Comment:
As we have discussed, this may change depending on our business requirements for registration. It is possible that our user will not have an account ID until completing a minimum step from the registration process. In this case, we will refactor the following route to exclude the ID param